chatbot/ai-studio.yaml

```diff
@@ -1,16 +1,31 @@
 applications:
-  # type: language
-  # name: chatbot
-  # description: This is a LLM chatbot application
+  type: language
+  name: chatbot
+  description: This is a LLM chatbot application
   containers:
-    - name: chatbot-example
-      contextdir: ai-application/builds
-      #image-name: quay.io/redhat-et/chatbot:latest
+    # podman run -p 7860:7860 quay.io/sallyom/model-services:llama2-7b-chatbot-arm
+    # these model-service images provide the gradio UI and a prompt to interact with the model
     - name: chatbot-model-arm
-      contextdir: model-services/builds/arm
+      contextdir: model-services/builds
       arch: arm64
       model-service: true 
+      #image-name: quay.io/sallyom/model-services:llama2-7b-chatbot-arm
     - name: chatbot-model-x86
-      contextdir: model-services/builds/x86
+      contextdir: model-services/builds
       arch: amd64
       model-service: true 
+      #image-name: quay.io/sallyom/model-services:llama2-7b-chatbot
+    - name: chatbot-model-x86-cuda
+      contextdir: model-services/builds/cuda
+      arch: amd64
+      model-service: true 
+      #image-name: quay.io/sallyom/model-services:llama2-7b-chatbot-cuda
+    - name: chatbot-example-arm
+      contextdir: ai_applications/builds
```

Comment:
for our sample apps, we don't need these `ai_applications/builds/Containerfile` images, the model-service images alone currently provide what is necessary for a simple sample app.

Reply:
I'm not suggesting we remove them, we'll document how to build/run these, but they don't need to be included in what we build as the sample app in `ai-studio.yaml`

Reply:
or, on 2nd thought, we can keep these in, because it will provide the "this is how you develop an application to interact with the model-service"